app/views/publisher/_publisher.mobile.haml

```diff
@@ -31,7 +31,7 @@
       %ul#photodropzone
     #fileInfo-publisher
 
-    #file-upload-publisher{title: t('shared.publisher.upload_photos'), class: 'btn btn-default'}
+    #file-upload-publisher{class: 'btn btn-default'}
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.